packages/foreman/rubygem-safemode/rubygem-safemode.spec

```diff
@@ -7,11 +7,13 @@ Summary: A library for safe evaluation of Ruby code
 Name: %{?scl_prefix}rubygem-%{gem_name}
 
 Version: 1.3.5
-Release: 3%{?dist}
+Release: 4%{?dist}
 Group: Development/Ruby
 License: MIT
 URL: https://github.com/svenfuchs/safemode
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+# Patch safemode to allow to_sentence for array
+Patch0: https://patch-diff.githubusercontent.com/raw/svenfuchs/safemode/pull/32.patch
```

Comment:
You're not annexing the patch, so this can be just `32.patch` or even better `allow-to_sentence.patch` ;)

Reply:
not blocking, as it works as it is

Reply:
I wanted to keep the URL so it is easily recognizable that the patch is upstream.